### ISSUE TITLE
proposal: remove @ from beginning of display names

### ISSUE THREAD

**Issue by mmckegg** [ssbc/patchwork#442]
Seriously considering just doing it. Speak up now if you're attached to them! Me and @ahdinosaur had a discussion about it last night.

This is definitely a matter of taste, and I was initially onboard with the prefix. However, I feel that as patchwork embraces full names (with spaces etc, as opposed to twitter style handles), it is starting to look odd. IMO it makes sense to treat name display more like facebook does.

It has also become a little confusing for users when they rename people. Sometimes they type the `@` which is just nasty (`@@mikey`).

`@` mentions would still work. Typing @ would bring up the suggest box, but **would not** prefix the resulting person's name in the link with an `@`.

Patchwork's user interface will highlight usernames in a way that makes them obvious that they are people (links to feeds), so there should be no confusion.

**Comment by mmckegg** [comment]
[porthole into cross-posted thread on ssb](http://viewer.scuttlebot.io/%25KUSJZDQk7nAAdVPP%2BiZRfG37O0Yp1VGD5dqmd%2F%2FcOjk%3D.sha256)

**TL;DR** controversy!

**Comment by mmckegg** [comment]
I think if I were to implement this, it would just be removing @ from sidebar buttons and the post headers. Everything else would keep working how it does now.
